### DISCUSSION TITLE
封面和扉页怎么加上去

### DISCUSSION THREAD

**Discussion by a1wj1** [sikouhjw/gdutthesis#86] [General]
你好，请问Latex的封面和扉页怎么加到pdf

**Comment by sikouhjw** [comment]
我认为你对 latex 及模板的了解有误区，请修正你的认识：
1. pdf 是 latex 编译的产物，没有『加到』pdf 这一说法
2. 模板直接编译就会生成中英扉页，你的问题不成立
3. 封面通过设置
    https://github.com/sikouhjw/gdutthesis/blob/f7e5fd504bb3c220d9708d9573630236ed0d793c/main.tex#L22-L25
    `style/cover={true}` 生成
4. 这些内容在[手册](https://github.com/sikouhjw/gdutthesis/releases/latest/download/gdutthesis.pdf)内都有提及，请阅读手册再使用
    https://github.com/sikouhjw/gdutthesis/blob/f7e5fd504bb3c220d9708d9573630236ed0d793c/user-manual/gdutthesis.dtx#L1026-L1031
5. latex 的正确拼写为 `LaTeX` 或全小写 `latex`
